tests/foreman/ui/test_contenthost.py

```diff
@@ -1399,10 +1399,29 @@ def test_content_access_after_stopped_foreman(session, vm, foreman_service_teard
 
     :Assignee: lpramuk
     """
-    result = vm.run(f'yum -y install {FAKE_1_CUSTOM_PACKAGE}')
+    sat = foreman_service_teardown
+    org = sat.api.Organization().create()
+    # adding remote_execution_connect_by_ip=Yes at org level
+    sat.api.Parameter(
+        name='remote_execution_connect_by_ip', value='Yes', organization=org.id
+    ).create()
+    lce = sat.api.LifecycleEnvironment(organization=org).create()
+    with sat:  # ensure the context section only uses this satellite
```

Comment:
if you are using content manager on the sat here, will it modify the hostname in settings to old satellite hostname? will we be not missing the foreman teardown step ? or I'm missing anything here.

Reply:
the teardown fixture uses the same satellite as a fixture requirement now

Reply:
I'm thinking the teardown fixture should contain the context manager then? Or are you intentionally trying to show a split between direct `sat.api` chains and helper functions?

Reply:
this pattern shows how you can just wrap a smaller subsection of a test with a Satellite context manager. This could also be useful when using multiple Satellites in the same test, switching between each's context.